benchmark/main.cpp

```diff
@@ -260,6 +260,8 @@ MBED_NOINLINE static int benchmark_md4()
 
     BENCHMARK_FUNC_CALL("MD4", mbedtls_md4_ret(buf, BUFSIZE, tmp));
 
+    ret = 0;
```

Comment:
While this code is correct, I find it confusing. I would consider setting the `ret`'s value inside the benchmark macro - it would express exactly what the commit's title says: "if feature is unavailable, return 0".